scr/nyctaxi/data_processor.py

```
@@ -0,0 +1,45 @@
+        self.config = config
+        self.preprocessor = None
+
+    def load_data(self, filepath):
+        #return pd.read_csv(filepath)
+        return self.spark.read.format("delta").load(filepath)
+
+    def preprocess_data(self):
+        # Remove rows with missing target
+        target = self.config['target']
+        self.df = self.df.dropna(subset=[target])
+        
```

Ruff

Blank line contains whitespace

Remove whitespace from blank line

(W293)